src/bin/rm/rm.rs

```diff
@@ -10,8 +10,8 @@ use std::path::PathBuf;
 #[derive(Debug, Args)]
 #[clap(version)]
 pub struct RmArgs {
-    /// Crates to be removed.
-    #[clap(value_name = "CRATE", required = true)]
+    /// Dependencies to be removed.
+    #[clap(value_name = "DEP_ID", required = true)]
     crates: Vec<String>,
```

Comment:
Note that the field name here remains unchanged, as is the case with cargo add here.